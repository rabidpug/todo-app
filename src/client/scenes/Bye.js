import React from 'react';
import logo from 'Assets/logo.svg';
import styles from './Bye.scss';

const Bye = () => (
  <div className={ styles.bye }>
    <img src={ logo } />
    {'Bye'}
  </div>
);

export default Bye;
