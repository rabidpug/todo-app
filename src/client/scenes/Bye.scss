.bye {
  color: #00f;
}
