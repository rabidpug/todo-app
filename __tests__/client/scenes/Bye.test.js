import Bye from 'Scenes/Bye';
import React from 'react';
import { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';

describe(
  'TopBar', () => {
    it(
      'Should render the TopBar', () => {
        const wrapper = shallow( <Bye /> );

        expect( shallowToJson( wrapper ) ).toMatchSnapshot();
      }
    );
  }
);
