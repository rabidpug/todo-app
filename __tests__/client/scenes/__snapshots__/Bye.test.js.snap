// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render the TopBar 1`] = `
<div
  className="bye"
>
  <img
    src="test-file-stub"
  />
  Bye
</div>
`;
